### DISCUSSION TITLE
troubleshooting steps

### DISCUSSION THREAD

**Discussion by pascaljoly** [Ribbit-Network/ribbit-network-frog-hardware#147] [Q&A]
if the sensor does not appear on the map, it is not clear what are the troubleshooting steps. could you point to the right direction?
the screenshot on https://github.com/Ribbit-Network/ribbit-network-frog-sensor/blob/main/hardware/v3/assembly-instructions/7-debugging.md is missing.

**Comment by keenanjohnson** [comment]
Hey @pascaljoly ! Ah yes sorry about that! It seems we didn't quite finish it from the V3 update. I will submit a fix now.

**Comment by keenanjohnson** [comment]
@pascaljoly I've fixed those images links here: https://github.com/Ribbit-Network/ribbit-network-frog-sensor/blob/main/hardware/v3/assembly-instructions/7-debugging.md
